### PR TITLE
Run e2e tests in CI

### PR DESCRIPTION
**Pull request by serathius**
Ensure that e2e with release tests are run.

### CODE REVIEW

**Comment by serathius** [comment]
cc @ahrtr @ptabor

File: tests/e2e/etcd_config_test.go [hunk 1]
```diff
@@ -16,18 +16,19 @@ package e2e
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/assert"
 	"io/ioutil"
 	"os"
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
```
Comment (jmhbnz) [inline tests/e2e/etcd_config_test.go:24]: Hey @serathius - Nitpick - when I run `go run golang.org/x/tools/cmd/goimports -w tests/e2e/etcd_config_test.go` on this it is suggesting the following:

```diff
 import (
        "fmt"
-       "github.com/stretchr/testify/assert"
        "io/ioutil"
        "os"
        "strings"
        "testing"
 
+       "github.com/stretchr/testify/assert"
+
        "go.etcd.io/etcd/pkg/expect"
 )
```

**Review by jmhbnz — APPROVED** [review]
LGTM

**Review by ahrtr — APPROVED** [review]
Please resolve @jmhbnz 's comment before merging this PR. thx

**Pull request merged**